### DISCUSSION TITLE
Any ETA on a new release?

### DISCUSSION THREAD

**Discussion by StoppingBuck** [iSoron/uhabits#1478] [Help & FAQ] [answered]
Sorry if this has already been asked before, but the latest release was in August 2021, but there seems to be a lot of active development since then.
Are there any plans for a new release soon? Or is there a feature roadmap somewhere to show how far off it is?

**Comment by iSoron** [comment, marked as answer]
Version 2.1 will be released when all the blockers identified in #1372 are fixed. There is no ETA.
